src/NHibernate.Test/DebugSessionFactory.cs

```diff
@@ -400,7 +401,7 @@ public static ISessionCreationOptions GetCreationOptions(IStatelessSessionBuilde
 				(ISessionCreationOptions)sessionBuilder;
 		}
 
-		internal class SessionBuilder : ISessionBuilder
+		internal class SessionBuilder : ISessionBuilder, ISessionCreationOptionsWithMultiTenancy
```

Comment:
Seems like `ISessionBuilder` needs to be an abstract/virtual class. Can you please add a TODO for 6.0?

Reply:
I don't get it. Where TODO should be added? This is test class wrapper. And `ISessionCreationOptionsWithMultiTenancy`  is a temp interface for supporting such custom interface implementations and it will be dropped in 6.0. My understanding no abstract class is required.

Reply:
Nevermind